picturefill-background.js

```diff
@@ -2,6 +2,13 @@
     "use strict";
 
     /**
+     * Utilities
+     */
+    var escapeRegExp = function( string ) {
+        return string.replace( /[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&' );
```

Comment:
`\W` will escape characters that don't need escaping, this method is lifted from [http://stackoverflow.com/a/6969486](http://stackoverflow.com/a/6969486) and could be made less verbose by only escaping the characters that are strictly necessary, à la [https://developer.mozilla.org/en/docs/Web/JavaScript/Guide/Regular_Expressions](https://developer.mozilla.org/en/docs/Web/JavaScript/Guide/Regular_Expressions):

```javascript
var escapeRegExp = function( string ) {
    return string.replace( /[.*+?^${}()|[\]\\]/g, '\\$&' );
}
```